coderd/database/dbauthz/dbauthz.go

```diff
@@ -1810,6 +1810,14 @@ func (q *querier) FetchVolumesResourceMonitorsUpdatedAfter(ctx context.Context,
 	return q.db.FetchVolumesResourceMonitorsUpdatedAfter(ctx, updatedAt)
 }
 
+func (q *querier) FindMatchingPresetID(ctx context.Context, arg database.FindMatchingPresetIDParams) (uuid.UUID, error) {
+	_, err := q.GetTemplateVersionByID(ctx, arg.TemplateVersionID)
```

Comment:
I don't like this, but I can't find a better way and there is precedent for this approach. Open to alternatives.

Reply:
The alternative is to essentially duplicate the logic in `GetTemplateVersionByID`, which I'm less a fan of.